internal/timeutil/location/loc_bench_test.go

```diff
@@ -0,0 +1,74 @@
+package location
+
+import (
+	"testing"
+	"time"
+)
+
+var loc *time.Location
+
+func BenchmarkGMT(b *testing.B) {
+	b.ReportAllocs()
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			GMT()
```

Comment:
Should we check if the returned object is nil or not? 🤔

Reply:
I think we shouldn't check the result as it should be well tested in unit test.
I think it is not the responsibility of benchmark test.
What do you think?

Reply:
I got it. But How do you know if the benchmark is working properly? 🤔 
we don't need to check the details of the object, but I thought it would be okay to check the return values ​​nil or not nil.

Reply:
And I thought that if I didn't store it in a variable, the compiler would be optimized and the result would be different from the actual use case. 🤔 
If I made a mistake, I am sorry.

Reply:
@vankichi what do you think about this from @hlts2?
`actual usecase stored it to variable.`

Reply:
it is ok.
I think we change only this line,
```suggestion
			l := GMT()
```

Reply:
@vankichi if we change that way, the compile error will be returned due to unused variable.
we may consider change it to:
```suggestion
        var l *time.Location
	b.RunParallel(func(pb *testing.PB) {
		for pb.Next() {
			l = GMT()
```
cc: @hlts2

Reply:
or this? 🤔 
```suggestion
			_ := GMT()
```

Reply:
@vankichi it is same as calling GMT() directly... 🤔
so it is `_ = GMT()` not `:=` 😵

Reply:
I think it is good to define a variable before `b.RunPrarallel`.
It seems to store same address 🤔

Reply:
yes it store to the same address.. 
if you think it is good, I will update like this.
```go
        var l *time.Location
	b.RunParallel(func(pb *testing.PB) {
		for pb.Next() {
			l = GMT()
```
cc: @hlts2

Reply:
@vankichi @hlts2 since there is compile error if we do not use `l` in the above code, I change the implementation like this. What do you think?
```
func BenchmarkGMT(b *testing.B) {
	var l *time.Location
	b.ReportAllocs()
	b.ResetTimer()
	b.RunParallel(func(pb *testing.PB) {
		for pb.Next() {
			l = GMT()
		}
	})
	_ = l
}
```

Reply:
@kevindiu 
In the first place, Is this code correct? 🤔 
Is it okay to rewrite the same variable from multiple Goroutines of RunParallel?

Reply:
I think the cost of writing the same variable is small and we can ignore it, we do not put any lock on it so it should be ok.
what do you think? @hlts2

Reply:
I just tied but the above code won't work well.
did you check it?